vision/module/region_of_interest.py

```diff
@@ -0,0 +1,74 @@
+"""
+region_of_interest will
+given the depth value (float) in millimeters
+
+The __main__ of this file acts as a driver for testing region_of_interest
+"""
+import numpy as np
+import argparse
+import matplotlib.pyplot as plt
+
+# Constant
+MODULE_HEIGHT = 76.2  # mm
+MODULE_WIDTH = 50.8  # mm
+VERTICAL_FOV = 57  # degrees?
+HORIZONTAL_FOV = 86  # degrees
+VERTICAL_RES = 1080  # p
+HORIZONTAL_RES = 1920  # p
+
+# ratio of the measured region_of_interest, use from 0 to 1
+#   this should be used to ensure that the region_of_interest is trustworthy despite tilt
+PADDING_CONSTANT = .85
+
+# TODO: account for general orientation (horizontal or vertical) and switch horizontal/vertical accordingly
+# keep in mind that, as of right now, the method assumes that the module is vertically oriented, and the best
+#   solution to account for it is to decrease the PADDING_CONSTANT
```

Comment:
Since we will calculate this before we calculate orientation, we will need the padding constant to work for all reasonable tilt values. We could potentially drop any outliers in the region though numpy ie
```python
outlier_mask = image > np.percentile(image, 90) & image < np.percentile(image, 10)
image = image[outlier_mask]
```
or
```python
image = np.clip(image, np.percentile(image, 10), np.percentile(image, 90))
```
Depending on speed.